src/DotNetProjectFile.Analyzers/MsBuild/NoWarn.cs

```diff
@@ -0,0 +1,9 @@
+namespace DotNetProjectFile.MsBuild;
+
+public sealed class NoWarn(XElement element, Node parent, MsBuildProject project)
+    : Node<string>(element, parent, project) 
+{
+    public IReadOnlyList<string> RuleIds
```

Comment:
Should we also trim after splitting? I'm not sure how whitespace is treated here by MSBuild

Reply:
It is more that we better detect the numbers correctly. So, yes, trimming is a good idea.